src/code.c

```diff
@@ -201,6 +180,24 @@ SEXP method_call_(SEXP call, SEXP generic, SEXP envir) {
     R7_method_lookup_error(generic, signature_classes, envir);
   }
 
+  // Now add the remaining, non-dispatched, args
+  SEXP gen_formals = FORMALS(generic);
+  R_xlen_t gen_formals_len = Rf_xlength(gen_formals);
+  if (gen_formals_len > gen_signature_len) {
+    for (R_xlen_t i = gen_signature_len; i < gen_formals_len; ++i) {
+      // Find value of that argument
+      SEXP name = Rf_install(CHAR(STRING_ELT(gen_signature_args, i)));
```

Comment:
What's the best way to get the names of the remaining arguments? (I'm just using `gen_signature_args` as a placeholder)